package.json

```
@@ -18,6 +18,7 @@
     "build": "turbo run build",
     "build:apps": "turbo run build --filter='@openmrs/*-app'",
     "setup": "yarn install && turbo run build",
+    "start": "openmrs develop --sources packages/apps/esm-*-app",
```

I'm always nervous about starting all the apps, so I'd probably just do:

```suggestion
    "start": "openmrs develop",
```